pkg/apk/expandapk/expandapk.go

```diff
@@ -175,13 +175,12 @@ func (m *multiReadCloser) Close() error {
 }
 
 func (a *APKExpanded) Close() error {
-	errs := []error{}
-
-	if a.tempDir != "" {
-		errs = append(errs, os.RemoveAll(a.tempDir))
-	}
-
-	return errors.Join(errs...)
+	// We cant' remove the tempDir because our cached files are advertised
+	// by symlinks pointing into them.
+	//
+	// This is not a big deal because the temp files if not referred by
+	// a symlink will be cleaned up anyway.
+	return nil
```

Comment:
There are other callers into this (I didn't expose this package) that do actually want to clean up the temp files, so I'd rather keep this the same (simplify to just `os.RemoveAll(a.tempDir)`) and remove the call to `Close()` inside of apko.

Reply:
Done. Thanks